### PR TITLE
Unify endpoints

### LINKED ISSUE #2700: Unify game links
There are 2 routes for the game page:

- /game/view/:game_id
- /game/:game_id

Similarly, demos and reviews also have a `/view/` route.

https://github.com/online-go/online-go.com/blob/devel/src/routes.tsx#L238-L246

Currently, Tournaments, Automatch and Incident reports link to the `/game/view/` route ([codesearch](https://github.com/search?q=repo%3Aonline-go%2Fonline-go.com+%2Fgame%2Fview%2F&type=code)), while the rest of the site links to the `/game/` route ([codesearch](https://github.com/search?q=repo%3Aonline-go%2Fonline-go.com+%2F%5C%2Fgame%5C%2F%5C%24%5C%7B%2F&type=code)).

While we should keep these routes for backwards compatibility, OGS should be consistent about how it links internally.

## Additional Context

https://forums.online-go.com/t/linking-to-a-specific-move/51913/6

### PR DESCRIPTION
**Pull request by LForchini**
Fixes #2700

## Proposed Changes

  - Replaces `/game/view` calls with `/game`
  - Replaces `/review/view` with `/review`
  - Replaces `/demo/view` with `/demo`
 
This did not impact functionality when I attempted to use the new routes.

### CODE REVIEW

File: src/routes.tsx [hunk 1]
```diff
@@ -235,13 +235,13 @@ export const routes = (
                 <Route path="/chat/:channel/**/*" element={<ChatView />} />
                 <Route path="/chat" element={<ChatRedirect />} />
                 <Route path="/observe-games" element={<ObserveGames />} />
-                <Route path="/game/view/:game_id" element={<Game />} />
+                <Route path="/game/:game_id" element={<Game />} />
```
Comment (anoek) [inline src/routes.tsx:238]: We need to keep the routes here for external links

**Comment by anoek** [comment]
Awesome thanks :) A little too thorough with the search and replace there, but non routes stuff I think is g2g

**Comment by LForchini** [comment]
I've reverted the changes in the router, is this the correct usage?

**Comment by anoek** [comment]
Looks great, thanks!

**Comment by benjaminpjones** [comment]
Thank you @LForchini!

**Pull request merged**